packaging/setup/ovirt_engine_setup/keycloak/constants.py

```diff
@@ -0,0 +1,73 @@
+#
+# ovirt-engine-setup -- ovirt engine setup
+#
+# Copyright oVirt Authors
+# SPDX-License-Identifier: Apache-2.0
+#
+#
+
+
+"""Constants."""
+
+
+import gettext
+import os
+
+from otopi import util
+
+from ovirt_engine_setup.constants import osetupattrsclass
+from ovirt_engine_setup.engine import constants as oenginecons
+from ovirt_engine_setup.engine_common import constants as oengcommcons
+
+from . import config
+
+def _(m):
+    return gettext.dgettext(message=m, domain='ovirt-engine-setup')
+
+
+@util.export
+@util.codegen
+class Const(object):
+    SERVICE_NAME = 'grafana-server'
+    OVIRT_ENGINE_KEYCLOAK_SETUP_PACKAGE_NAME = \
+        'ovirt-engine-keycloak'
+
+
+@util.export
+@util.codegen
+@osetupattrsclass
+class ApacheEnv(object):
+    HTTPD_CONF_OVIRT_ENGINE_KEYCLOAK = 'OVESETUP_APACHE/configFileOvirtEngineKeycloak'
+
+
+@util.export
+@util.codegen
+class FileLocations(oengcommcons.FileLocations):
```

Comment:
You can base on ovirt_engine_setup.constants.FileLocations and then have some of the below "for free"